Source/Core/DiscIO/DiscExtractor.cpp

```diff
@@ -130,13 +130,15 @@ void ExportDirectory(const Volume& volume, const Partition& partition, const Fil
                      const std::string& export_folder,
                      const std::function<bool(const std::string& path)>& update_progress)
 {
-  File::CreateFullPath(export_folder + '/');
+  std::string export_root =
```

Comment:
You should be able to make this `const` as well.

Reply:
good catch. done